website/data/docs-navigation.js

```diff
@@ -324,8 +324,12 @@ export default [
                   'standalone-tls',
                   'standalone-audit',
                   'ha-with-consul',
+                  'ha-with-raft',
                   'external',
                   'kubernetes-auth',
+                  'enterprise-with-raft.mdx',
+                  'enterprise-dr-with-raft.mdx',
+                  'enterprise-perf-with-raft.mdx',
```

Comment:
Are these supposed to have a `.mdx` suffix? The netlify jobs seem to be failing with logs like this:
```
2:02:13 PM: Error: The page "enterprise-with-raft.mdx" was not found within the category 
"platform/k8s/helm/examples". Please double-check to ensure that 
"docs/platform/k8s/helm/examples/enterprise-with-raft.mdx.mdx" exists. If this page was never 
intended to exist, remove the key "enterprise-with-raft.mdx" from the category 
"platform/k8s/helm/examples" in "data/docs-navigation.js"
```

Reply:
Good catch, let me see if I can fix it.

Reply:
Removed `.mdx`, hopefully it fixes it.